doc/manual/functions.rst

```diff
@@ -210,12 +210,17 @@ Functions in Julia are `first-class objects
 variables, called using the standard function call syntax from the
 variable they have been assigned to. They can be used as arguments, and
 they can be returned as values. They can also be created anonymously,
-without being given a name:
+without being given a name, using either of these sintax:
```

Comment:
@ivarne sorry I won't try to fix anything again when it's late at night and without spell checker :sweat: